src/test/java/net/minecraftforge/debug/world/LoginPacketSplitTest.java

```diff
@@ -66,6 +66,7 @@
  * To test if the packet is too large simply remove the login packet from the {@link net.minecraftforge.network.filters.ForgeConnectionNetworkFilter}
  * and try connecting again. You should see the connection fail.
  */
+/**
```

Comment:
@Matyrobbrt why u cause me pain.

Please fix the compile errors and reinstate this test mod by:
- Changing the call to `FriendlyByteBuf#writeWithCodec` into `FriendlyByteBuf#writeJsonWithCodec`; and
- Changing the reference to `com.mojang.bridge.game.PackType.DATA` to simply `PackType.SERVER_DATA`, importing the new class.